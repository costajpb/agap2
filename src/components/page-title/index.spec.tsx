import { render } from '@testing-library/react'
import PageTitle from '.'

describe('components/page-title', () => {
    it('should render', () => {
        const { container } = render(<PageTitle>Page title</PageTitle>)
        expect(container).toMatchSnapshot()
    })
})
