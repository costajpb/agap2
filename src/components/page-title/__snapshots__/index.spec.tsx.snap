// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/page-title should render 1`] = `
<div>
  <h1
    class="sc-cWSHoV dKuyNJ"
  >
    <span
      class="sc-kMkxaj jWvPnq"
    >
      Page title
    </span>
  </h1>
</div>
`;
